deployment/E_heroku.md

```diff
@@ -1 +1,277 @@
-## FPO
+### What we'll need
+
+- a working application
+
+### Goals
+
+Our main goal for this is guide is to get a Phoenix application running on Heroku.
+
+## Steps
```

Comment:
What would you think about adding a quick sentence here to lead into the list of sections? Iirc, there's a sentence in the Exrm Releases Guide that fulfills this purpose.